docs/Makefile

```diff
@@ -0,0 +1,180 @@
+# Makefile for Sphinx documentation
```

Comment:
I always feel this Makefile is kinda superfluous: we probably should just do the call to sphinx-build and that would be easier: we don't use the other targets.